utils/logging.ts

```diff
@@ -0,0 +1,13 @@
+import { Context } from "@azure/functions";
+
+export const mapWithPrivacyLog = (
+  context: Context,
+  logPrefix: string,
+  stringToObfuscate?: string
```

Comment:
Can we use a `NonEmptyString` type?
```suggestion
  stringToObfuscate?: NonEmptyString
```